_includes/payment-order-ssn-restrictions.md

```diff
@@ -0,0 +1,458 @@
+{% capture api_resource %}{% include api-resource.md %}{% endcapture %}
+{% capture documentation_section %}{% include documentation-section.md %}{%
+endcapture %}
+{% capture features_url %}{% include documentation-section-url.md href='/features' %}{% endcapture %}
+
+## Restrict payments to a Social Security Number
+
+Swedbank Pay provides the possibility to restrict payments to a Social Security Number for payment instruments which support this. This can be used when you want to make sure you only accept payment from an already identified individual. You do this by adding the field `restrictedToSocialSecurityNumber` in the Payer node, in your payment order request, and setting it to true. This will leave out all instruments which do not support this feature. It will then use the `socialSecurityNumber` located in the `nationalIdentifier` node which is located in the `payer` node. `nationalIdentifier` must exist to be able to use this feature. Instruments supporting the feature will reject payments that do not match the restriction.
+
+{% if documentation_section contains "checkout-v3/enterprise" %}
+If you want to only use the SocialSecurityNumber to restrict a payment and not do a lookup to checkout profile, add the parameter `guestMode` in the `nationalIdentifier` node and set it to `true`. {% endif %}
+
+You are currently only able to restrict Swish and Trustly payments to a Social Security Number, but we will add support for more payment instruments going forward. No changes are required at your (the merchant’s) end to be able to offer more instruments at a later time.
+
+## Restrict to Social Security Number Request
+
+The field itself is a `bool` which must be added in the `payer` node of the request. Below is an example of a Payment Order request.
```

Comment:
The text looks great! I think we can leave out everything else than the affected nodes in the example to make the relevant fields more visible, and the same with the tables. See the Custom Logo feature for an example.

Reply:
Just tweak the text to say "Apart from the new field, the payment request is similar to a basic one" or something along those lines. A link to the relevant request is also nice :)